### PR TITLE
cnv2068 - viewing IP addresses of additional networks

### PR DESCRIPTION
**Pull request by aburdenthehand**
Small assembly with two modules. Once the QEMU guest agent is installed and running on the VM, the IP address of additional networks is passed to the host and displayed in the VM Details screen of the web console, or in a query to describe the vmi on the CLI.

https://jira.coreos.com/browse/CNV-2068

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@openshift/team-documentation PTAL

File: modules/cnv-viewing-vmi-ip-web.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+= Viewing the IP address of a virtual machine interface in the web console
+
+The IP information displays in the *Virtual Machine Details* screen for the virtual machine. 
+
+.Procedure
+
+. In the {ProductName} console, click *Workloads* -> *Virtual Machines*.
+. Click the virtual machine name to open the *Virtual Machine Details* screen.
+
+The information for each attached vNIC is displayed under *IP ADDRESSES*.
```
Comment (mburke5678) [inline modules/cnv-viewing-vmi-ip-web.adoc]: Do we need an example output here, maybe with call outs to describe what is shown?

File: cnv/cnv_users_guide/cnv-viewing-ip-of-vm-vnic.adoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[id="cnv-viewing-ip-of-vm-vnic"]
+= Viewing the IP address of vNICs on a virtual machine
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-viewing-ip-of-vm-vnic
+toc::[]
+
+The QEMU guest agent runs on the virtual machine and passes the IP address of attached vNICs to the host, allowing you to view it from both the web console and the `oc` client.
```
Comment (mburke5678) [inline cnv/cnv_users_guide/cnv-viewing-ip-of-vm-vnic.adoc]: What is `it`?

File: modules/cnv-viewing-vmi-ip-cli.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+//
+// * cnv_users_guide/cnv-viewing-ip-of-vm-vnic.adoc
+
+[id="cnv-viewing-vmi-ip-cli_{context}"]
+= Viewing the IP address of a virtual machine interface in the CLI
+
+The network interface configuration is included in the `oc describe vmi` command.
+
+You can also view the IP address information by running `ip addr` on the virtual
+machine, or by running `oc get vmi -o yaml`.
```
Comment (mburke5678) [inline modules/cnv-viewing-vmi-ip-cli.adoc]: Where to run `oc get vmi -o yaml`? Master?

**Comment by mburke5678** [comment]
@aburdenthehand A few things. Maybe in the context of the CNV docs you can ignore. Otherwise LGTM.

File: modules/cnv-viewing-vmi-ip-cli.adoc (hunk 3 above)
Comment (aburdenthehand) [reply modules/cnv-viewing-vmi-ip-cli.adoc]: Anywhere that has the client installed.
We have a user story for the 1.4 migration designed around covering the CLI tools, to cover things like `oc login`, but it doesn't exist here yet.

File: modules/cnv-viewing-vmi-ip-web.adoc (hunk 1 above)
Comment (aburdenthehand) [reply modules/cnv-viewing-vmi-ip-web.adoc]: It's just the IP addresses against the vNIC name displayed in the web console with other VM details.

File: cnv/cnv_users_guide/cnv-viewing-ip-of-vm-vnic.adoc (hunk 2 above)
Comment (aburdenthehand) [reply cnv/cnv_users_guide/cnv-viewing-ip-of-vm-vnic.adoc]: Mike Patton would be proud :)
But seriously, I've updated with s/it/the IP address/'

**Comment by aburdenthehand** [comment]
Thanks for the review @mburke5678!

File: cnv/cnv_users_guide/cnv-viewing-ip-of-vm-vnic.adoc (hunk 2 above)
Comment (mburke5678) [reply cnv/cnv_users_guide/cnv-viewing-ip-of-vm-vnic.adoc]: We care a lot...

**Comment by aburdenthehand** [comment]
@mburke5678 - This should be right to merge - thank you!

**Pull request merged**